contribs/gmf/src/datasource/Manager.js

```diff
@@ -488,9 +488,11 @@ const exports = class {
     const name = gmfLayer.name;
     const timeAttributeName = meta.timeAttribute;
     const visible = meta.isChecked === true;
+    const columnsOrder = meta.columnsOrder;
```

Comment:
We should add the `columnsOrder` property definition to the `gmfThemes.GmfMetaData` in `gmf-themes.js`